src/targets/python/requests/client.test.ts

```diff
@@ -0,0 +1,17 @@
+import { runCustomFixtures } from '../../../fixtures/runCustomFixtures';
+import { Request } from '../../../httpsnippet';
+
+runCustomFixtures({
+  targetId: 'python',
+  clientId: 'requests',
+  tests: [
+    {
+      fixtureFile: 'query-params.py',
+      options: {
+        showBoilerplate: false,
+      },
+      request: { method: 'GET', url: 'http://mockbin.com/har?param=value' } as Request,
+      title: "should support query parameters provided in HAR's url",
```

Comment:
Want to move this to the top of the test object so it's consistent with how the others are set up?

Reply:
good feedback: did so in 6f6a18835035894b3fb5ef5a4d97b5c29761651c and made sure they were all the same while I was at it.